presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -747,6 +781,39 @@ private void testCreateInvalidBucketedTable(HiveStorageFormat storageFormat)
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testInsertSortedBucketedTable()
+    {
+        String tableName = "test_insert_sorted_bucketed_table";
+
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE " + tableName + " " +
+                "(id int, name varchar, ds varchar) " +
+                "WITH (" +
+                "bucketed_by = ARRAY[ 'id' ], " +
+                "bucket_count = 11, " +
+                "sorted_by = ARRAY[ '+name' ]" +
+                ")";
+        assertUpdate(createTable);
+
+        try {
+            assertUpdate(
+                    "INSERT INTO " + tableName + " " +
+                            "VALUES " +
+                            "  (1, VARCHAR 'b', VARCHAR 'c'), " +
+                            "  (2, 'bb', 'cc'), " +
+                            "  (3, 'bbb', 'ccc')",
+                    3);
+            fail("expected failure");
+        }
+        catch (Exception e) {
+            assertEquals(e.getMessage(), format("Inserting into bucketed sorted tables is not supported. Table 'tpch.%s'", tableName));
```

Comment:
If the message is not what you expect (e.g. NPE with null message), the exception raised by the test will be hardly useful. Use `org.assertj.core.api.Assertions#assertThatThrownBy`:
```
assertThatThrownBy(() -> assertUpdate(...))
  .hasMessage(...)
```